Tests/SwiftFusionTests/Geometry/Pose2Tests.swift

```diff
@@ -356,4 +356,37 @@ final class Pose2Tests: XCTestCase {
       }
     }
   }
+
+  /// Tests group action: translate only
+  func testActionTranslate() {
+    let x = Vector2(1.0, -2.0)
+    let pose = Pose2(-3.0, 4.0, 0.0)
+
+    let expected = Vector2(-2.0, 2.0)
+    let actual = pose * x
+
+    assertAllKeyPathEqual(actual, expected, accuracy: 1e-9)
+  }
+
+  /// Tests group action: rotate only
+  func testActionRotate() {
+    let x = Vector2(1.0, 2.0)
+    let pose = Pose2(0.0, 0.0, 0.5)
+
+    let expected = Vector2(-0.08126851531803325,  2.2345906623849485)
```

Comment:
Make tests easy and obvious :-) Use an angle like 30 or 60 degrees for which cos and sine are well known. Write sqrt() if you need to. don't use 45 as its special.

Reply:
Got it, will do.

Reply:
Done.